test/livebook/zta/global_test.exs

```diff
@@ -0,0 +1,16 @@
+defmodule Livebook.ZTA.GlobalTest do
+  use ExUnit.Case, async: true
+
+  test "ZTA provider setting tests" do
```

Comment:
Let's add this to `test/livebook/config_test.exs`.

Reply:
sounds good, will add it there